hack/images.yaml

```diff
@@ -20,3 +20,4 @@ zookeeper-0.2.13:
   - "docker.io/pravega/zookeeper-operator:0.2.13"
   - "docker.io/pravega/zookeeper:0.2.13"
   - "docker.io/pravega/zookeeper:0.2.12"
+  - "docker.io/lachlanevenson/k8s-kubectl:v1.16.10"
```

Comment:
this adds to the gazillion versions of `docker.io/lachlanevenson/k8s-kubectl` we have so far. :see_no_evil: 
no need to change, just a note to cleanup in the future.

Reply:
yeah I know 😢, I could override with a version we already include in the other image bundle but then we have to worry about keeping them in sync 🤔